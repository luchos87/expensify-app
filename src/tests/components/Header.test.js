import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';

test('should render Header corectly', ()  => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
});
