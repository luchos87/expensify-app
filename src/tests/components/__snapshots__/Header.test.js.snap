// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header corectly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/dashboard"
  >
    home   
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    create  
  </NavLink>
  <button
    onClick={[Function]}
  >
    Logout
  </button>
</header>
`;
